### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by nbraud**
- [x] Add @ppb/maintainers as the default owner.
- [x] Experiment with assigning code to @astronouth7303 and myself.

### CODE REVIEW

**Comment by pathunstrom** [comment]
That's the typical 3.8 failure, right?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default owner is the maintainer team
+* @ppb/maintainers
+
+# @nbraud owns the Hypothesis testsuite
```
Comment (pathunstrom) [inline .github/CODEOWNERS]: I like this experiment. If it works here in Vector, I will probably do some of this regarding bits of the core engine.

**Comment by nbraud** [comment]
@pathunstrom I hadn't noticed the Py3.8 failure earlier, but I just made a patch for it, so this can wait for #139 to get in

**Comment by nbraud** [comment]
bors try

**Comment by nbraud** [comment]
@pathunstrom The 3.8-rc builds (and the others) succeeded (see the green checkmark next to the “Try #138” commit for a complete list).  I guess this is ready to ship now  :)

**Comment by astronouth7303** [comment]
bors retry

**Comment by nbraud** [comment]
Rude  :o

**Comment by nbraud** [comment]
bors retry

**Pull request merged**